components/atom/toast/src/index.scss

```diff
@@ -0,0 +1,134 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import './settings';
+
+.sui-AtomToast {
+  background-color: $bgc-atom-toast;
+  border-radius: $bdrs-atom-toast;
+  border: $bd-atom-toast;
+  box-shadow: $bxsh-atom-toast;
+  color: $c-atom-toast-text;
+  font-size: $fz-atom-toast;
+  overflow: hidden;
+  position: fixed;
+  z-index: $z-notifications;
+
+  &-icon {
+    cursor: pointer;
+    margin: $m-s;
+    position: absolute;
+    right: 0;
+
+    svg {
+      fill: $c-atom-toast-icon !important;
+      height: $sz-icon-s;
+      width: $sz-icon-s;
+    }
+  }
+
+  // Margin
+  &-margin {
+    &--s {
+      margin: $m-s;
+    }
+
+    &--m {
+      margin: $m-m;
+    }
+
+    &--l {
+      margin: $m-l;
+    }
+  }
```

Comment:
Please remove these options, and replace with a component token (value = $m-m)
Ticket also adjusted